### PR TITLE
Update journal-of-historical-linguistics.csl

### PR DESCRIPTION
**Pull request by damnation333**
via error request: https://forums.zotero.org/discussion/comment/271743/#Comment_271743 
It should come out as In Hall PA & Soskice D, eds., (comma between names and label), but comes out as In Hall PA & Soskice D eds.,

Am I missing something?

### CODE REVIEW

File: journal-of-historical-linguistics.csl [hunk 1]
```diff
@@ -28,7 +28,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name delimiter-precedes-last="always" initialize="false" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+      <name delimiter-precedes-last="always" initialize="false" initialize-with="." name-as-sort-order="first"/>
```
Comment (adam3smith) [inline journal-of-historical-linguistics.csl]: this should have `and="symbol"`

File: journal-of-historical-linguistics.csl [hunk 2]
```diff
@@ -111,15 +111,13 @@
           <text macro="title" prefix=" "/>
           <group prefix=".">
             <group>
-              <group suffix=".">
-                <names variable="editor translator">
-                  <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always" prefix=" In: " suffix=","/>
+              <group prefix=" " suffix=".">
+                <text term="in" text-case="capitalize-first" suffix=" "/>
+                <names variable="editor translator" delimiter="," suffix=",">
+                  <name delimiter=" " and="symbol" delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
```
Comment (adam3smith) [inline journal-of-historical-linguistics.csl]: I'm not seeing initials in the bibliographies, so remove `initialize-with=""`. I also see names in regular order, so remove the `name-as-sort-order`. Finally, I'm not seeing a comma before the ampersand, so set `delimiter-precedes-last="never"`
I'm working off this example:
image
Might also be there are different ones? But this makes sense in the context of the style

Comment (adam3smith) [inline journal-of-historical-linguistics.csl]: That same example also suggests that there shouldn't be an "in" and that the label should be before the name and be `verb-short`.

File: journal-of-historical-linguistics.csl [hunk 3]
```diff
@@ -111,15 +111,13 @@
...
                   <label form="short" prefix=" "/>
                 </names>
-                <text variable="collection-title" prefix=" " suffix="." font-style="italic"/>
-                <text variable="container-title" prefix=" " suffix="." font-style="italic"/>
-                <text macro="publisher" prefix=" " suffix=", "/>
-                <text variable="page" suffix="."/>
+                <text variable="page" prefix=" " suffix="."/>
```
Comment (adam3smith) [inline journal-of-historical-linguistics.csl]: it looks like we're missing the publisher macro here now?

**Comment by rmzelle** [comment]
> It should come out as In Hall PA & Soskice D, eds., (comma between names and label), but comes out as In Hall PA & Soskice D eds.,
> 
> Am I missing something?

You'd have to change the "label" element in

```xml
        <else-if type="chapter paper-conference" match="any">
          <text macro="title" prefix=" "/>
          <group prefix=".">
            <group>
              <group prefix=" " suffix=".">
                <text term="in" text-case="capitalize-first" suffix=" "/>
                <names variable="editor translator" delimiter="," suffix=",">
                  <name delimiter=" " and="symbol" delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
                  <label form="short" prefix=" "/>
                </names>
                <text variable="page" prefix=" " suffix="."/>
              </group>
            </group>
          </group>
        </else-if>
```

to `<label form="short" prefix=", "/>`.

**Comment by damnation333** [comment]
Thanks for reviewing this style. That chapter section was completely messed up. sorry.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**